requirements.txt

```
@@ -1,6 +1,6 @@
 httplib2==0.9.2; python_version <= '2.7'
 httplib2==0.19.0; python_version > '3'
-pysimplesoap==1.08.14
+git+https://github.com/pysimplesoap/pysimplesoap.git@stable_py3k#pysimplesoap
```

This is breaking py2.7 install, please use a conditional dependency:
```suggestion
pysimplesoap==1.08.14; python_version <= '2.7'
git+https://github.com/pysimplesoap/pysimplesoap.git@stable_py3k#pysimplesoap; python_version > '3'
```